prometheus/common/alert-rules.d/crunchy-alert-rules-etcd.yml.example

```diff
@@ -10,11 +10,58 @@ groups:
 
 ########## ETCD EXPORTER RULES ##########
 
+#### IMPORTANT NOTE: These alerts only support monitoring a single etcd cluster per promemtheus instance ####
```

Comment:
```suggestion
#### IMPORTANT NOTE: These alerts only support monitoring a single etcd cluster per prometheus instance ####
```

Reply:
What happens if you need additional monitored clusters? Is there a workaround?

Reply:
There is currently no way to distinguish which etcd system is part of which etcd cluster through the metrics they provide. @sharmay had been trying to look at that and hadn't found anything so far.

Reply:
Okay, at least it's documented; will consider this as addressed as we can.

Reply:
Actually, just thought of a way that might work, but would require manual setup by the user.

The `etcd_name` label could be given a patterned name per cluster. Then an alert could be configured with a wildcard based on that name.